INSTALL

```diff
@@ -64,260 +64,10 @@ WHAT DO YOU NEED ?
-   dependent (.cmx, .o and executable files for instance) but others are not
-   (.cmo and .vo). Consequently, you can :
-
-   o  save some time during compilation by not cleaning the architecture
-      independent files;
-
-   o  save some space during installation by sharing the Coq standard
-      library (which is fully architecture independent).
-
-   So, in order to compile Coq for a new architecture, proceed as follows:
-
-   * Omit step 7 above and clean only the architecture dependent files:
-     it is done automatically with the command
-
-		make archclean
-
-   * Configure the system for the new architecture:
-
-		./configure <options>
-
-     You can specify the same directory for the standard library but you
-     MUST specify a different directory for the binaries (of course).
-
-   * Compile and install the system as described in steps 5 and 6 above.
-
-
-MOVING BINARIES OR LIBRARY.
-===========================
-
-   If you move both the binaries and the library in a consistent way,
-   Coq should be able to still run. Otherwise, Coq may be "lost",
-   running "coqtop" would then return an error message of the kind:
-
-	Error during initialization :
-	Error: cannot guess a path for Coq libraries; please use -coqlib option
-
-   You can then indicate the new places to Coq, using the options -coqlib :
-
-	coqtop -coqlib <new directory>
-
-   See also next section.
-
-
-DYNAMICALLY LOADED LIBRARIES FOR BYTECODE EXECUTABLES.
-======================================================
-
-   Some bytecode executables of Coq use the OCaml runtime, which dynamically
-   loads a shared library (.so or .dll). When it is not installed properly, you
-   can get an error message of this kind:
-
-	Fatal error: cannot load shared library dllcoqrun
-	Reason: dllcoqrun.so: cannot open shared object file: No such file or directory
+Coq offers the choice of two build systems, a Dune-based one, and a
+make-based one.
 
-   In this case, you need either:
-     - to set the CAML_LD_LIBRARY_PATH environment variable to point to the
-       directory where dllcoqrun.so is; this is suitable when you want to run
-       the command a limited number of times in a controlled environment (e.g.
-       during compilation of binary packages);
-     - install dllcoqrun.so in a location listed in the file ld.conf that is in
-       the directory of the standard library of OCaml;
-     - recompile your bytecode executables after reconfiguring the location
-       of the shared library:
-         ./configure -vmbyteflags "-dllib,-lcoqrun,-dllpath,<path>" ...
-       where <path> is the directory where the dllcoqrun.so is installed;
-     - (not recommended) compile bytecode executables with a custom OCaml
-       runtime by using:
-         ./configure -custom ...
-       be aware that stripping executables generated this way, or performing
-       other executable-specific operations, will make them useless.
+Please see INSTALL.make.md for specific instructions.
```

Comment:
No recommendation at all for where to look for specific instructions for Dune?

Reply:
This was done in the next commit of the "set dune as default" PR, however it is a good idea to cherry pick that too [with different wording]